docs/README.md

```diff
@@ -16,7 +16,7 @@ CometBFT serves blockchain applications.
 
 More formally, CometBFT performs Byzantine Fault Tolerant (BFT)
 State Machine Replication (SMR) for arbitrary deterministic, finite state machines.
-For more background, see [What is CometBFT?](introduction/README.md#what-is-cometbft.md).
+For more background, see [What is CometBFT?](./introduction/README.md#what-is-cometbft.md).
```

Comment:
```suggestion
For more background, see [What is CometBFT?](./introduction/README.md).
```

Reply:
The anchor here is the page title, so we should remove it.

Reply:
I will keep it for now just to ensure it will not break in the future (e.g. a new section is added before that). And since the link says "What is CometBFT" I'd think it's better to leave as is